### PR TITLE
General form validation

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/views/PageUserProfile.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+    <div>
+        <TemplateUserProfile></TemplateUserProfile>
```
Comment (arrogant-ant) [inline src/views/PageUserProfile.vue:3]: creating a template with just one component.
should we have broken it differently?

File: src/components/organisms/OrganismKycForm.vue [hunk 2]
```diff
@@ -106,4 +123,25 @@ export default {
 };
 </script>
 
-<style></style>
+<style lang="scss" scoped>
+.kyc-form-header {
+    background-color: var(--secondary-color);
+    padding: 20px 30px;
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismKycForm.vue:129]: done

File: src/components/organisms/OrganismKycForm.vue [hunk 3]
```diff
@@ -106,4 +123,25 @@ export default {
...
+    margin-bottom: 30px;
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismKycForm.vue:130]: done

File: src/components/organisms/OrganismKycForm.vue [hunk 4]
```diff
@@ -106,4 +123,25 @@ export default {
...
+    }
+
+    h2 {
+        color: #e8faff;
+        font-size: 14px;
+    }
+}
+
+.kyc-form-main {
+    padding: 0 30px;
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismKycForm.vue:145]: done

File: src/components/organisms/OrganismLogin.vue [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        direction: ltr;
+        font-weight: 500;
+        height: auto;
+        line-height: normal;
+        min-height: 40px;
+        padding: 8px 16px;
+        margin-bottom: 50px;
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismLogin.vue:115]: done

File: src/components/molecules/MoleculeModal.vue [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+};
+</script>
+
+<style lang="scss">
+.modal-close {
+    display: none !important;
+}
+
+.modal-background {
+    background-color: rgb(10 10 10 / 15%) !important;
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/molecules/MoleculeModal.vue:41]: yes needed ...I want to override any existing css used

File: src/components/organisms/OrganismParkingFacility.vue [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                landmark: '',
+            },
+        };
+    },
+    methods: {
+        updateParkingSpotImg(data) {
+            this.model.parkingSpotImg = data;
+        },
+
+        updateAmenitiesData(amenitiesData) {
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismParkingFacility.vue:96]: yes for later development

File: src/router/routes.js [hunk 8]
```diff
@@ -52,6 +52,16 @@ export const routes = [
         name: 'Home',
         component: Home,
     },
+    {
+        path: '/user/profile',
+        name: 'userProfile',
+        component: () => import('@/views/PageUserProfile.vue'),
+    },
+    {
+        path: '/user/edit-profile',
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/router/routes.js:61]: done

File: src/store/user/index.js [hunk 9]
```diff
@@ -1,6 +1,32 @@
...
+import {
+    signInWithPopup,
+    GoogleAuthProvider,
+    signOut,
+    onAuthStateChanged,
+} from 'firebase/auth';
 
 const state = {
+    user: null,
+    userProfile: {
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/store/user/index.js:13]: will remove once all api integrated

File: src/views/PageUserProfile.vue (hunk 1 above)
Comment (alias01) [reply src/views/PageUserProfile.vue:3]: this page is removed

File: src/components/organisms/OrganismLogin.vue [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+                        <img
+                            src="../../assets/googleicon.svg"
+                            alt="gmail icon"
+                        />
+                    </span>
+                    <span class="google-btn-text"> Sign in With Google </span>
+                </button>
+                <div class="login-footer">
+                    <p>
+                        By continuing, you are indicating that you accept our
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismLogin.vue]: Suggestion?

[the remark this answers is not in the document]
Comment (arrogant-ant) [reply src/components/organisms/OrganismLogin.vue]: add href

**Review by arrogant-ant — APPROVED** [review]

File: src/components/organisms/OrganismLogin.vue (hunk 10 above)
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/OrganismLogin.vue:27]: done

**Pull request merged**